### PR TITLE
[WIP]ROC-2506 document management integration

### PR DESCRIPTION
**Pull request by robertparkinson**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -192,3 +194,74 @@ services:
...
+        - CORS_ORIGIN_METHODS=GET,POST,PUT,DELETE,OPTIONS
+        - CORS_ORIGIN_WHITELIST=http://evidence-management-api:8080
+        #      logging env vars
+        - ROOT_APPENDER=JSON_CONSOLE
+        - JSON_CONSOLE_PRETTY_PRINT=false
+        - REFORM_SERVICE_TYPE=node
+        - REFORM_SERVICE_NAME=em-api-gateway-web
+        - REFORM_TEAM=cc
+        - REFORM_ENVIRONMENT=docker
+        - XFWD=true
```
Comment (timja) [inline docker-compose.yml]: what's this for?

File: docker-compose.yml [hunk 2]
```diff
@@ -192,3 +194,74 @@ services:
...
+        - SPRING_PROFILES_ACTIVE=dev
+        - IDAM_TESTING_SUPPORT_ENABLED=true
+        - LOGGING_LEVEL_UK_GOV_HMCTS_IDAM=DEBUG
+        - IDAM_CLIENT_URL=http://idam-api:8080
+        - PROVIDER_SERVICE_CLIENT_URL=http://service-auth-provider-api:8080
+        - http_proxy=
+        - https_proxy=
+        - no_proxy=
+  #      logging env vars
+        - ROOT_APPENDER=JSON_CONSOLE
```
Comment (timja) [inline docker-compose.yml:258]: you probably don't want json console in local environment, I would remove all the logging config form here except the logging level

File: docker-compose.yml [hunk 3]
```diff
@@ -192,3 +194,74 @@ services:
         retries: 20
     smtp-server:
       image: mailhog/mailhog
+    document-management-api-gateway-web:
+      image: docker.artifactory.reform.hmcts.net/evidence/em-api-gateway-web:latest
```
Comment (timja) [inline docker-compose.yml:209]: when you are done remember to lock the version in

**Comment by rlewan** [comment]
Moved to https://github.com/hmcts/cmc-integration-tests/pull/11

**Pull request closed without merging**